Cargo.toml

```
@@ -1,27 +1,28 @@
 [workspace]
 members = [
-    "node",
-    "primitives",
-    "runtime",
-    "zrml/authorized",
-    "zrml/court",
-    "zrml/liquidity-mining",
-    "zrml/market-commons",
-    "zrml/orderbook-v1",
-    "zrml/orderbook-v1/fuzz",
-    "zrml/prediction-markets",
-    "zrml/prediction-markets/fuzz",
-    "zrml/prediction-markets/runtime-api",
-    "zrml/rikiddo",
-    "zrml/rikiddo/fuzz",
-    "zrml/simple-disputes",
-    "zrml/swaps",
-    "zrml/swaps/fuzz",
-    "zrml/swaps/rpc",
-    "zrml/swaps/runtime-api",
+  "node",
+  "primitives",
+  "runtime",
+  "zrml/authorized",
+  "zrml/court",
+  "zrml/liquidity-mining",
+  "zrml/market-commons",
+  "zrml/orderbook-v1",
+  "zrml/orderbook-v1/fuzz",
+  "zrml/prediction-markets",
+  "zrml/prediction-markets/fuzz",
+  "zrml/prediction-markets/runtime-api",
+  "zrml/rikiddo",
+  "zrml/rikiddo/fuzz",
+  "zrml/simple-disputes",
+  "zrml/swaps",
+  "zrml/swaps/fuzz",
+  "zrml/swaps/rpc",
+  "zrml/swaps/runtime-api",
+  "zrml/styx",
 ]
 resolver = "2"
 
 [patch."https://github.com/purestake/moonbeam"]
-pallet-author-mapping = { git = "https://github.com/zeitgeistpm/moonbeam", branch = "downgrade-staking-and-mapping" }
-parachain-staking = { git = "https://github.com/zeitgeistpm/moonbeam", branch = "downgrade-staking-and-mapping" }
+pallet-author-mapping = {git = "https://github.com/zeitgeistpm/moonbeam", branch = "downgrade-staking-and-mapping"}
+parachain-staking = {git = "https://github.com/zeitgeistpm/moonbeam", branch = "downgrade-staking-and-mapping"}
```

It's standard style to have whitespace between curly braces and their content.

```suggestion
pallet-author-mapping = { git = "https://github.com/zeitgeistpm/moonbeam", branch = "downgrade-staking-and-mapping" }
parachain-staking = { git = "https://github.com/zeitgeistpm/moonbeam", branch = "downgrade-staking-and-mapping" }
```

---

Why doesnt `cargo fmt` respect this?

---

Oh actually I might have done a format on this from my editor so nm